source/pkcore/abilitypages/chi/rapidpunch.rst

```diff
@@ -0,0 +1,64 @@
+.. rapidpunch:
+###########
+RapidPunch
+###########
+
+Description
+###########
+
+RapidPunch is a Chi Ability.
+
+RapidPunch allows the Chiblocker to punch rapidly in a short period, dealing massive damage. GIF TO BE ADDED
```

Comment:
Change to `RapidPunch allows Chiblockers to punch an opponent multiple times in a short period, dealing massive damage. GIF TO BE ADDED`